src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+import java.util.SplittableRandom;
+
+public class Deadline extends Task {
+    protected String by;
```

Comment:
I like that your class variables are never declared public, in accordance to the coding standards 👍